app/account/views.py

```diff
@@ -132,6 +112,25 @@ def change_password():
     return render_template('account/manage.html', form=form)
 
 
+@account.route('/manage/change-account-info', methods=['GET', 'POST'])
+@login_required
+def change_account_info():
+    """
+    Change an existing user's account information (excluding email and
+    password).
+    """
+    form = ChangeAccountInfoForm()
+    if form.validate_on_submit():
+        current_user.first_name = form.first_name.data
+        current_user.last_name = form.last_name.data
+        db.session.add(current_user)
+        db.session.commit()
+    else:
+        form.first_name.data = current_user.first_name
+        form.last_name.data = current_user.last_name
```

Comment:
Added a comment - "When form is first displayed, pre-populate its fields with the user's current information."